### PR TITLE
Bump current project and marketing version to 0.5.0.

### LINKED ISSUE #174: Prepare for release `0.4.0`
## Depends on tasks:
- #9 
- #59 
- #140

## Description:
Prepare the documentation and source for release.

## User requirements:
User should be able to go to the release page in repo, get the information about changes that were done and be able to download/setup the app

## Dev requirements and restrictions:

- Create a new branch for the release with the name "release-0.4.0" and set it up for release;
- Create a new branch to bump the version, if requires.
- Check the CHANGELOG.md, verify that there are no missed changes, and update with release information;
- Add the necessary info to the [Release_notes](https://github.com/ios-course/ironfoudation-team-project/blob/main/release_notes.md) in a user-friendly format based on CHANGELOG.md.
- Before merging to the main branch set up the tag "v.0.4.0" on the last commit before the merge commit.
- Publish the Release under the Release [section on GitHub.](https://github.com/ios-course/ironfoudation-team-project/releases)

### PR DESCRIPTION
**Pull request by kirpichonok**
## Summary
This PR is a part of the work to resolve #174.

## Changes
- Bump `Current Project Version` of the project, `Woof` target and `WoofTests` target to 0.5.0.
- Bump `Marketing Version` of the project, `Woof` and `WoofTests` targets to 0.5.0.

## Checks to complete
- [x] The PR title has a conventional commit style (short and descriptive).
- [x] The PR description includes essential information that helps understand the purpose of the changes.
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] I've updated `CHANGELOG.md` file.
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
  - Update revision of the `SimpleNetworkService` package in the project - [#204](https://github.com/ios-course/ironfoudation-team-project/pull/204)
  - Update `SitterProfileView` to allow the user to enter profile info only once - [#212](https://github.com/ios-course/ironfoudation-team-project/pull/212)
  - Update `SitterListView` to present a loading indicator and occurring errors - [#218](https://github.com/ios-course/ironfoudation-team-project/pull/218)
+ - Bump `Current Project Version` and `Marketing Version` to 0.5.0 - [#219](https://github.com/ios-course/ironfoudation-team-project/pull/219)
```
Comment (iBYarchitect) [inline CHANGELOG.md:31]: Do we need this note?
We don't have any notes about bumping the project version.

**Review by iBYarchitect — COMMENTED** [review]
I have one question about the changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (kirpichonok) [reply CHANGELOG.md:31]: Do you think it's not needed?

Comment (NadzeyaShpakouskaya) [reply CHANGELOG.md:31]: Agree with @iBYarchitect.

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM, after fixing changelog

**Pull request merged**